cmd/plume/prune.go

```diff
@@ -30,9 +32,12 @@ import (
 )
 
 var (
-	days        int
-	pruneDryRun bool
-	cmdPrune    = &cobra.Command{
+	days              int
+	dayssoftdeleted   int
+	keeplast          int
```

Comment:
camelCase, please. (If it appears as a double, sorry, the last one I wrote disappeared after a refresh).

Reply:
ok